development-vm/README.md

```diff
@@ -37,6 +37,8 @@ First, install:
 * [VirtualBox](https://www.virtualbox.org/)
 * [Vagrant](https://www.vagrantup.com/downloads.html)
 * vagrant-dns plugin: `vagrant plugin install vagrant-dns`
+* install vagrant: `vagrant dns --install`
```

Comment:
Great that we're improving the docs! 👍 

Could we indent these two new lines, so it's more obvious that they relate to the vagrant-dns plugin?
Also 'install vagrant' and 'start vagrant' do not describe these steps accurately. https://github.com/BerlinVagrant/vagrant-dns has some more accurate descriptions